draft-ounsworth-pq-composite-sigs.mkd

```diff
@@ -77,7 +77,7 @@ informative:
 --- abstract
 With the widespread adoption of post-quantum cryptography will come the need for an entity to possess multiple public keys on different cryptographic algorithms. Since the trustworthiness of individual post-quantum algorithms is at question, a multi-key cryptographic operation will need to be performed in such a way that breaking it requires breaking each of the component algorithms individually. This requires defining new structures for holding composite public keys and composite signature data.
```

Comment:
just a comment: post-quantum is a use case and a driving force. but this draft is quite generic and it seems to me that it can be used for any signature scheme. we are actually +1 for adoption because it maybe can be used as a tool that makes experimentation easier